<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerWatchdogModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerWatchdogModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>556</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>556</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Watchdog node: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="ModuleNodeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLWatchdogNode</string>
        </stringlist>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="selectNodeUponCreation">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Observed nodes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="qMRMLNodeComboBox" name="ToolComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLTransformNode</string>
            <string>vtkMRMLVolumeNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddToolButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="ToolsTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="VisibilityLabel">
          <property name="text">
           <string>&amp;Visible:</string>
          </property>
          <property name="buddy">
           <cstring>VisibilityCheckBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="VisibilityCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="DisplayNodeViewLabel">
          <property name="text">
           <string>View:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="qMRMLDisplayNodeViewComboBox" name="DisplayNodeViewComboBox">
          <property name="toolTip">
           <string>Select views in which to show this node. All unchecked shows in all 3D and 2D views.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="OpacityLabel">
          <property name="text">
           <string>Opacity:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ctkSliderWidget" name="OpacitySliderWidget">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="pageStep">
           <double>0.250000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="ColorLabel">
          <property name="text">
           <string>Background color</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ctkColorPickerButton" name="BackgroundColorPickerButton">
          <property name="color">
           <color>
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
          <property name="displayColorName">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="EdgeColorLabel">
          <property name="text">
           <string>Text Color:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="ctkColorPickerButton" name="TextColorPickerButton">
          <property name="color">
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="FontSizeLabel">
          <property name="text">
           <string>Font size</string>
          </property>
          <property name="buddy">
           <cstring>FontSizeSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="FontSizeSpinBox">
          <property name="suffix">
           <string> pt</string>
          </property>
          <property name="minimum">
           <number>6</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerWatchdogModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ToolComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerWatchdogModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ModuleNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
